Formula/gron.rb

```diff
@@ -37,6 +37,11 @@ class Gron < Formula
     :revision => "f344a1ffbe51794516e9cf2c4d58b203863d3070"
   end
 
+  go_resource "github.com/pkg/errors" do
+    url "https://github.com/pkg/errors.git",
+    :revision => "17b591df37844cde689f4d5813e5cea0927d8dd2"
```

Comment:
You mean Tim 😉 ? Actually, pythonists care the least about white space, because it's pretty much dictated by the language to begin with :) The fiercest debates I have seen about white space happen in C and Haskell.